packages/turbo/src/VisitableView.tsx

```diff
@@ -107,11 +115,40 @@ const VisitableView = React.forwardRef<RefObject, React.PropsWithRef<Props>>(
     );
 
     const handleOnOpenExternalUrl = useCallback(
-      (e: NativeSyntheticEvent<OpenExternalUrlEvent>) =>
-        onOpenExternalUrlCallback(e.nativeEvent),
+      ({ nativeEvent }: NativeSyntheticEvent<OpenExternalUrlEvent>) =>
+        onOpenExternalUrlCallback(nativeEvent),
       [onOpenExternalUrlCallback]
     );
 
+    const handleAlert = useCallback(
+      ({ nativeEvent: { message } }: NativeSyntheticEvent<AlertHandler>) => {
+        if (onAlert) {
+          onAlert(message);
+        } else {
+          Alert.alert(message);
+        }
+      },
+      [onAlert]
+    );
+
+    const handleConfirm = useCallback(
+      ({ nativeEvent: { message } }: NativeSyntheticEvent<AlertHandler>) => {
+        const dispatch = (value: boolean) =>
+          dispatchCommand(
+            visitableViewRef,
+            'sendConfirmResult',
+            value.toString()
+          );
+        if (onConfirm) {
+          onConfirm(message, (value) => dispatch(value));
+        } else {
+          Alert.alert('Confirm dialog', message);
```

Comment:
We probably want the default behavior to show the message as the title and with two buttons; `OK` and `Cancel`